src/core_ocean/Registry.xml

```diff
@@ -2608,7 +2682,7 @@
 		<var_array name="vertNonLocalFlux" type="real" dimensions="nVertLevelsP1 nCells Time" packages="forwardMode;analysisMode">
 			<var name="vertNonLocalFluxTemp" array_group="vertNonLocalFlux" units="nondimensional" name_in_code="vertNonLocalFluxTemp"
 				 description="CVMix/KPP: nonlocal boundary layer mixing term for temperature"
-		/>
+			/>
```

Comment:
I added a tab here because, in other `var_array` environments, the closing parenthesis of `var` aligns with their opening parenthesis. The closing parenthesis for `var_array` is `</var_array>` which is aligned with `<var_array ...>` later in the code.

Reply:
Oh, I see.  YOu are right.